include/aspect/particle/world.h

```diff
@@ -317,10 +306,10 @@ namespace aspect
         std_cxx11::unique_ptr<Output::Interface<dim> > output;
 
         /**
-         * Set of particles currently in the local domain, organized by
-         * the level/index of the cell they are in.
+         * Particle handler object that is responsible for storing and
+         * managing the internal particle structures.
          */
-        std::multimap<types::LevelInd, Particle<dim> > particles;
+        std_cxx11::unique_ptr<ParticleHandler<dim> > particle_handler;
```

Comment:
there is nothing wrong with making it a pointer. but out of curiosity, why not make the particle handler a regular member variable?